src/Cake.Common/Build/TFBuild/Data/TFBuildPublishTestResultsData.cs

```diff
@@ -23,7 +23,7 @@ public class TFBuildPublishTestResultsData
         /// <summary>
         /// Gets or sets the list of Test Result files to publish.
         /// </summary>
-        public string[] TestResultsFiles { get; set; }
+        public IEnumerable<FilePath> TestResultsFiles { get; set; }
```

Comment:
Can we change this one to an `ICollection<FilePath>` and initialize it?

```
public ICollection<FilePath> TestResultsFiles { get; set; } = new List<DirectoryPath>();
```

Reply:
Fixed.  It didn't like `List<DirectoryPath>` so I used `List<FilePath>` instead. 😄